faunadb-java/src/main/java/com/faunadb/client/query/Language.java

```diff
@@ -256,16 +312,34 @@ public static Value Create(Value ref) {
   /**
    * Creates a new Create function.
    *
-   * <p><b>Reference</b>: <a href="https://faunadb.com/documentation#queries-write_functions">FaunaDB Write Functions</a></p>
+   * <p><b>Reference</b>: <a href="https://faunadb.com/documentation/queries#write_functions">FaunaDB Write Functions</a></p>
    */
   public static Value Create(Value ref, Value params) {
     return ObjectV("create", ref, "params", params);
   }
 
   /**
+   * Creates a new Insert function.
```

Comment:
Use "expression" in these descriptions: "Creates a new _ function." -> "Creates a new _ expression." Should be backported to existing docs.